src/components/Agreements/ViewAgreement/ViewAgreement.js

```diff
@@ -42,7 +45,10 @@ class ViewAgreement extends React.Component {
     selectedAgreement: {
       type: 'okapi',
       path: 'erm/sas/:{id}',
-      shouldRefresh,
+      shouldRefresh: (resource, action, refresh) => {
+        const { path } = action.meta;
+        return refresh || (path && path.match(/link/));
```

Comment:
What's this do?

Reply:
It basically reloads the _selectedAgreement_ resource. _path && path.match(/link/)_ checks if https://github.com/folio-org/stripes-erm-components/pull/40/files#diff-b846f11013fdd8210fb6bff3fe8b7e2cR24_ is getting mutated and if it is it returns _true_ triggers the refresh

Reply:
Ahhh, I kept trying to figure out what _path_ had "link" in it and couldn't figure it out. It's the path _template_ that's getting matched. Good to know!